scripts/tests/sqa/test_sqa_check.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python
+import unittest
+
+class Test(unittest.TestCase):
+    def testSQAScript(self):
+        """Test the SQA checker."""
+
+
+        self.assertTrue(False)
```

Comment:
This file is named the same as the other one and will always fail...

Reply:
That guy is not supposed to be there.